src/org/zaproxy/zap/extension/api/ContextAPI.java

```diff
@@ -141,14 +171,46 @@ public ApiResponse handleApiAction(String name, JSONObject params) throws ApiExc
 	            throw new ApiException(ApiException.Type.NO_ACCESS, f.getAbsolutePath());
             } else {
             	try {
-					Model.getSingleton().getSession().exportContext(ctx, f);
+					Model.getSingleton().getSession().exportContext(context, f);
 				} catch (Exception e) {
 		            throw new ApiException(ApiException.Type.INTERNAL_ERROR, e.getMessage());
 				}
             }
-        } else {
-            throw new ApiException(ApiException.Type.BAD_ACTION);
+        case ACTION_INCLUDE_TECHS:
+        	context = ApiUtils.getContextByParamId(params, CONTEXT_ID);
+        	techSet = context.getTechSet();
+        	techNames = getParam(params, PARAM_TECH_NAMES, "").split(",");
+        	for (String techName : techNames) {
+        		techSet.include(getTech(techName));
+        	}
+        	context.setTechSet(techSet);

```

Comment:
This statement can be removed, it's redundant, the changes are already done to the _TechSet_ of the _Context_.
The same comment applies to cases _ACTION_EXCLUDE_TECHS_ and _ACTION_EXCLUDE_ALL_TECHS_.